### ISSUE TITLE
Excutable Missing

### ISSUE THREAD

**Issue by richard42** [mupen64plus/mupen64plus-user-issues#392]
Originally reported on Google Code with ID 393

```
Describe your system:
 - Operating System (be specific):Ubuntu Linux 10.10 Maverick 
 - Machine type (32-bit or 64-bit):32-bit
 - Mupen64Plus version:1.99.3
- Plugins used:

For visual artifacts, give:
- game name and CRC/MD5 hash (printed on console)
- description of the artifact (missing polygons, flashing, wrong colors,
etc)

Describe the problem: Using the instructions from the read me files I installed Mupen64Plus
1.99.3, but for some reason the executable is missing from usr/local/bin (I dont have
a bin folder). (So far I only have version 1.5). I have already installed 1.99.3 multiple
times and yet the executable is not put there.


Please provide any additional information below.
Mind putting the new version of Mupen64Plus in the Ubuntu Repos (Ubuntu Software Center)
because it only has the old v1.5 in the repos. 

```

Reported by `elementartes` on 2010-10-16 20:25:27

**Comment by richard42** [comment]
```
I can also confirm this issue with mupen 1.99.3 installer on Ubuntu 10.04 amd64. 

Some kids on the ubuntu forum [not me! also seem to be having problems: http://ubuntuforums.org/showthread.php?t=1357611
]

I've come to the same conclusion as them. 

In the setup script, it tries to install stuff in /usr/local/bin/ but nothing ends
up there.

These are the lines in install.sh that configure that.

PREFIX="${1:-/usr/local}"
SHAREDIR="${2:-${PREFIX}/share/mupen64plus}"
BINDIR="${3:-${PREFIX}/bin}"

I suspect Ubuntu probably wants the files to be dropped in a different location rather
than just "mkdir -p"'ing the locations but I'm not really sure where to be honest.

my hacking around:


timmy@speed-freed:/tmp/test/mupen64plus-bundle-linux64-1.99.3$ sudo bash install.sh

Installing Mupen64Plus Binary Bundle to /usr/local
install: creating directory `/usr/local/share/mupen64plus'
install: creating directory `/usr/local/share/mupen64plus/doc'
install: omitting directory `doc/emuwiki-api-doc'
timmy@speed-freed:/tmp/test/mupen64plus-bundle-linux64-1.99.3$ ls /usr/local/bin/
ls: cannot access /usr/local/bin/: No such file or directory
timmy@speed-freed:/tmp/test/mupen64plus-bundle-linux64-1.99.3$ ls /usr/local/lib/
libmupen64plus.so.2  libmupen64plus.so.2.0.0  python2.6  site_ruby
timmy@speed-freed:/tmp/test/mupen64plus-bundle-linux64-1.99.3$ sudo bash uninstall.sh

Uninstalling Mupen64Plus Binary Bundle from /usr/local
rmdir: failed to remove `/usr/local/bin': No such file or directory
timmy@speed-freed:/tmp/test/mupen64plus-bundle-linux64-1.99.3$ mkdir -p /usr/local/bin
mkdir: cannot create directory `/usr/local/bin': Permission denied
timmy@speed-freed:/tmp/test/mupen64plus-bundle-linux64-1.99.3$ sudo mkdir -p /usr/local/bin
timmy@speed-freed:/tmp/test/mupen64plus-bundle-linux64-1.99.3$ sudo bash install.sh

Installing Mupen64Plus Binary Bundle to /usr/local
install: creating directory `/usr/local/share/mupen64plus'
install: creating directory `/usr/local/share/mupen64plus/doc'
install: omitting directory `doc/emuwiki-api-doc'
timmy@speed-freed:/tmp/test/mupen64plus-bundle-linux64-1.99.3$ ls /usr/local/bin/
timmy@speed-freed:/tmp/test/mupen64plus-bundle-linux64-1.99.3$ 

```

Reported by `timmydobson` on 2010-10-17 21:54:35

**Comment by richard42** [comment]
```
Yea I would really like this to be fixed. The old Mupen64Plus wont let me change my
PS3 controller settings and wont even make it work half the time. Could the devs just
update the version in the repos to 1.99.3?
```

Reported by `elementartes` on 2010-10-21 22:40:55

**Comment by richard42** [comment]
```
ppa:sven-eckelmann/ppa-mupen64plus

And no, Debian will not get 1.99.3 and Ubuntu only fetches it from Debian.
```

Reported by `sven@narfation.org` on 2010-10-21 23:32:32

**Comment by richard42** [comment]
```
elementartes wrotes:
> How do I find the launcher for the Graphical Interface of Mupen64Plus?

There is no graphical interface in mupen64plus 1.99.x - please see http://code.google.com/p/mupen64plus/wiki/ThirdPartyPlugins
for some options
```

Reported by `sven@narfation.org` on 2010-10-23 14:44:40

**Comment by richard42** [comment]
```
This is fixed in 1.99.4.  Was caused by install script quitting after the 'omitting
directory...' message.
```

Reported by `richard42g` on 2010-11-29 13:14:54
- **Status changed**: `Fixed`

**Comment by richard42** [comment]
```
Thank you. :D
```

Reported by `timmydobson` on 2010-12-04 23:30:57
